src/webapp/vos_data.py

```diff
@@ -31,6 +31,10 @@ def get_tree(self, authorized=False, filters: Filters = None) -> Dict:
         for vo_name in sorted(self.vos.keys(), key=lambda x: x.lower()):
             try:
                 expanded_vo_data = self._expand_vo(vo_name, authorized=authorized, filters=filters)
+
+                # Remove DataFederations block from VOSummary
+                expanded_vo_data.pop('DataFederations', None)
```

Comment:
This needs to be inside the `if expanded_vo_data` block to avoid a potential TypeError if expanded_vo_data is None.

Reply:
Got this fixed in #299 also.  Just removed this hunk.